src/server/dfly_main.cc

```diff
@@ -186,7 +186,10 @@ bool RunEngine(ProactorPool* pool, AcceptServer* acceptor) {
 
   error_code ec = acceptor->AddListener(bind_addr, port, main_listener);
 
-  LOG_IF(FATAL, ec) << "Could not open port " << port << ", error: " << ec.message();
+  if (ec) {
```

Comment:
what about flushing the log? have you found it?

Reply:
I found the flush function and than I found that LOG(ERROR) does send to log before exit but I just didnt see it sometimes  because its not the last log line so I missed it. Therefor there is no need to call flush